src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -177,6 +177,10 @@ public function getEnv(string $prefix, string $name, \Closure $getEnv)
         }
 
         if (null === $env) {
+            if ('' === $prefix) {
```

Comment:
`''` becomes a magic value that acts as the `string` prefix except for null where we return null